recipes/forwarder.rb

```diff
@@ -21,11 +21,18 @@
 end
 node.set['logstash_forwarder']['config']['network']['servers'] = forwarder_servers
 
+# set lumberjack key locations and perms
+node.default['lumberjack']['ssl_dir'] = '/etc'
+node.default['lumberjack']['ssl_key_path'] = "#{node['lumberjack']['ssl_dir']}/#{node['lumberjack']['ssl key']}"
+node.default['lumberjack']['ssl_cert_path'] = "#{node['lumberjack']['ssl_dir']}/#{node['lumberjack']['ssl certificate']}"
+node.default['lumberjack']['user'] = 'root'
+node.default['lumberjack']['group'] = 'root'
```

Comment:
Maybe these could also be `default_unless`? Just so wrappers can easily override them.